default.nix

```diff
@@ -116,7 +116,7 @@ let
       (commonLib.pkgsDefault.callPackage ./overlays/rust/mozilla.nix {})
       (import ./overlays/rust)
     ];
-    pkgs = import sources.nixpkgs {
+    pkgs = import sources."nixpkgs-19.09" {
```

Comment:
When can we get rid of that?

Reply:
I guess when we bump Jormungandr :)

Reply:
I think `nixpkgs-mozilla` rust overlay is not compatible with 20.03. Did not dig deep into the error though.